code/game/machinery/computer/computer.dm

```diff
@@ -73,19 +73,18 @@
 	flickering = FALSE
 
 /obj/machinery/computer/update_icon()
-	overlays.Cut()
-	if((stat & NOPOWER) || force_no_power_icon_state)
-		if(icon_keyboard)
-			overlays += image(icon,"[icon_keyboard]_off",overlay_layer)
+	. = ..()
```

Comment:
we don't have this system at all

Reply:
Oh. Back to the drawing board, I guess. It doesn't work properly, anyway. The glowing screens still draw on top of people no matter how I try to set it up.